README.md

```diff
@@ -113,12 +113,23 @@ script:
 stages:
   - split
 
-services:
-  - docker
-
 split:
+  image: jderusse/gitsplit
   stage: split
+  cache:
+    key: "$CI_JOB_NAME/$CI_COMMIT_REF_NAME"
+    paths:
+      - /cache/gitsplit
+  variables:
+    GIT_STRATEGY: clone
+  before_script:
+    - eval $(ssh-agent -s)
+    - mkdir -p ~/.ssh
+    - '[[ -f /.dockerenv ]] && echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config'
```

Comment:
Not sure to be honest, it was part of every issue and documentation I found on the subject. It seems to be related only to legacy LXC containers. I'll strip it out.